### PR TITLE
[pRO] Login Packets

### PR DESCRIPTION
**Pull request by alisonrag**

### CODE REVIEW

File: src/Network/Send/ServerType0.pm [hunk 1]
```diff
@@ -196,7 +196,7 @@ sub new {
 		'025C' => ['auction_info_self', 'v', [qw(type)]],
 		'025D' => ['auction_sell_stop', 'V', [qw(ID)]],
 		'0273' => ['mail_return', 'V Z24', [qw(mailID sender)]],
-		'0275' => ['game_login', 'a4 a4 a4 v C x16 v', [qw(accountID sessionID sessionID2 userLevel accountSex iAccountSID)]],
+		'0275' => ['game_login', 'a4 a4 a4 v C Z16 V', [qw(accountID sessionID sessionID2 userLevel accountSex mac iAccountSID)]],
```
Comment (sctnightcore) [inline src/Network/Send/ServerType0.pm]: miss mac in sub?

Comment (alisonrag) [reply src/Network/Send/ServerType0.pm]: mac is useless seems not used by the server.
you can send any info server will accept

File: src/Network/Send/ServerType0.pm [hunk 2]
```diff
@@ -293,6 +293,7 @@ sub new {
 		'0A46' => ['stylist_change', 'v6' ,[qw(hair_color hair_style cloth_color head_top head_mid head_bottom)]],
 		'0A49' => ['private_airship_request', 'Z16 v' ,[qw(map_name nameID)]],
 		'0A6E' => ['rodex_send_mail', 'v Z24 Z24 V2 v v V a* a*', [qw(len receiver sender zeny1 zeny2 title_len body_len char_id title body)]],   # -1 -- RodexSendMail
+		'0A76' => ['master_login', 'V Z40 a32 v', [qw(version username password_rijndael master_version)]],
```
Comment (sctnightcore) [inline src/Network/Send/ServerType0.pm]: username not a32?

Comment (alisonrag) [reply src/Network/Send/ServerType0.pm]: There is no place in source where u can find a "username" as ASCII null-padded(a*).
I followed the pattern of all the other "usernames", i even followed the "old" game_login packet pattern

Comment (alisonrag) [reply src/Network/Send/ServerType0.pm]: 0A76 always had the length 80
using 32 as size to username you can't reach the total length

Comment (alisonrag) [reply src/Network/Send/ServerType0.pm]: image

**Pull request merged**